stacker/actions/diff.py

```diff
@@ -4,11 +4,10 @@
 import sys
 from operator import attrgetter
 
-import yaml
-
 from .base import plan, build_walker
 from . import build
 from .. import exceptions
+from ..awscli_yamlhelper import yaml_parse
```

Comment:
Could we put this in `cloudformation` module and name the method `parse_cloudformation_template` or something along the lines of that? I realize this mostly comes from how the AWS CLI named it, but I think we can name it a little better.

```python
from ..cloudformation import parse_cloudformation_template
```

Reply:
I think a cloudformation module or a function in util.py does make sense. Might be nice to have it wrap the vendored awscli_yamlhelper module to keep the vendored file as distinct/pristine as possible?